eng/targets/Imports.BeforeArcade.targets

```diff
@@ -17,11 +17,11 @@
   <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
     <KnownFrameworkReference Update="Microsoft.NETCore.App">
       <TargetingPackVersion Condition="'%(TargetFramework)' == 'net6.0'">6.0.0</TargetingPackVersion>
-      <TargetingPackVersion Condition="'%(TargetFramework)' == 'net8.0'">8.0.0-preview.3.23165.3</TargetingPackVersion>
+      <TargetingPackVersion Condition="'%(TargetFramework)' == 'net8.0'">$(MicrosoftNETCoreAppRefPackageVersion)</TargetingPackVersion>
```

Comment:
I'm not seeing that this defined in the repo.  Where is it defined?  This needs to be defined locally for the repo's source-build leg.  For the source-build product build it get's defined by the previously built packages.

Reply:
Yeah - I think I need to define these in `Versions.props`. Should we use the values specified above or pick up the latest, used in VMR?

Reply:
@mmitche is it OK to just define these properties in `Versions.props` for now? Per-repo prebuilts work needs to clean this up, otherwise properties will be locked to an old version.

Reply:
Should define this in both version.details.xml and versions.props.

Reply:
> Should define this in both version.details.xml and versions.props.

OK, this is now causing a build break due to a mismatch in `roslyn` compiler versions of the analyzer coming from `runtime` and the compiler that is included in toolset SDK:

`❌CSC(0,0): error CS9057: The analyzer assembly '/__w/1/s/artifacts/source-build/self/package-cache/microsoft.netcore.app.ref/8.0.0-preview.3.23165.3/analyzers/dotnet/cs/Microsoft.Interop.LibraryImportGenerator.dll' references version '4.6.0.0' of the compiler, which is newer than the currently running version '4.5.0.0'.`

@mmitche do you know how this can be resolved? Updating toolset SDK in this repo would only fix it temporarily - this would break again when new mismatch happens.

Reply:
Actually...back up a second, is this block needed any longer? Now that roslyn builds after the runtime, the ref pack should be provided by the runtime build, correct?

There may still be an error though. I wonder if this requires a re-bootstrap.

Reply:
Yes, runtime build should provide the refpack, but if we bring `runtime` dependency, we also bring some analyzers that were built in runtime, with `4.6.0` compiler.

Or, is the idea to remove this block altogether **and** not introduce `runtime` dependency? That would bring refpack from the SDK.

Reply:
> Or, is the idea to remove this block altogether and not introduce runtime dependency? That would bring refpack from the SDK.

This would be the closest to what we do in the product build. Would require PVP flow to be turned on What do you think @MichaelSimons ?

Reply:
Updated this PR with a simple solution @mmitche proposed.

cc @MichaelSimons @mmitche